lara-typescript/src/interactive-api-client/api.ts

```diff
@@ -175,6 +185,10 @@ export const setHeight = (height: number | string) => {
   getClient().post("height", height);
 };
 
+export const decoratedContentEvent = (msg: IDecoratedContentEventMessage) => {
+  getClient().post("decoratedContentEvent", msg);
+};
```

Comment:
The name `decoratedContentEvent` makes me think it _is_ an event, as opposed to a function which sends an event. Perhaps `postDecoratedContentEvent()`?

Reply:
I like this name better.  I was a bit uncertain about a few of the name choices - I think this makes things clearer.  Fixed.